webclient/vypis/urls.py

```diff
@@ -1,9 +1,11 @@
 from django.urls import path
 
-from .views import VypisView
+from .views import VypisListView, VypisOnlyView, VypisView
 
 app_name = "vypis"
 
 urlpatterns = [
     path("<str:typ_vazby>/<str:ident_cely>/", VypisView.as_view(), name="vypis"),
+    path("cisty/<str:typ_vazby>/<str:ident_cely>/", VypisOnlyView.as_view(), name="vypis_only"),
+    path("list/<str:typ_vazby>/<str:identy_cele>/", VypisListView.as_view(), name="vypisy"),
```

Comment:
- `<str:identy_cele>` - jak to bude v praxi vypadat? Nebude to moc dlouhé a nemělo by to tak jít přes POST?
- Je to tak, že vyhodnocení oprávnění pro dílčí záznamy stačí řešit pro `vypis` a `vypis_only` a zde se jakoby podědí pro dílčí záznamy ze seznamu? Nebo jak je to řešeno? (draft nastavení je v https://docs.google.com/spreadsheets/d/1it_kr3HCN2vv54SUvn696XaP9tKCgcHnqUCBeg4CGYw/edit?gid=0#gid=0)

Reply:
stranka samotna musi byt GET. pokud bychom to chteli delat pres POST pro dlzku, byla by to o dost vetsi komplexita. na 100 zaznamu to bude stacit jak to je. Myslim ze i vic to zvladne v pohode. Pokud by trebalo predelal bych to v budoucnu
ad2 presne jak pises. staci nastavit vypis a vypis_only